### ISSUE TITLE
SignatureCertChainUrl in testing?

### ISSUE THREAD

**Issue by Torsten85** [nickclaw/alexa-ability-express-handler#1]
Hi there,
this might be out of the scope of this package, but after implementing this in my https express server, I constantly get the error "No SignatureCertChainUrl header provided."

Doesn't amazon send this header in testing or did I something wrong?

Thanks in advance

**Comment by nickclaw** [comment]
Would you mind logging the headers you're receiving? You can add something like this to the top of your express app.

```js
app.use(function(req, res, next) {
  console.log(req.headers);
  next();
});
```

**Comment by Torsten85** [comment]
@nickclaw 
Hey, I now see the problem: The headers are all lowercase! Currently I have to do something like this to get it working:

```javascript
app.use(function(req, res, next) {
  const { signaturecertchainurl, signature } = req.headers;

  if (signaturecertchainurl) {
    req.headers.SignatureCertChainUrl = signaturecertchainurl;
  }

  if (signature) {
    req.headers.Signature = signature;
  }
  
  next();
});
```

**Comment by nickclaw** [comment]
That'll do it! Thanks for helping debug.

If you want to make a PR that uses the case insensitive `req.get(header)` instead, I can deploy it as a new version. Otherwise I'll get to it myself sometime this weekend.

**Comment by nickclaw** [comment]
Published as v0.4.4! @Torsten85 can you confirm the fix works?

**Comment by Torsten85** [comment]
@nickclaw Thank you, its working now as expected. But currently I'm facing a different problem, but I'll make an other issue for that.
